### PR TITLE
Update action examples to install svn

### LINKED ISSUE #151: ubuntu-latest lo longer comes with SVN installed
**Describe the bug**
[ubuntu-latest](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md) no longer has SVN installed, meaning example files will not function as expected.

**Steps to Reproduce**
Use ubuntu-latest in your .yml and you'll see "svn: command not found"

**Expected behavior**
The plugin should be deployed to SVN.

**Additional context**
Please recommend that authors use "ubuntu-22.04" instead in your example files.

### PR DESCRIPTION
**Pull request by thrijith**
### Description of the Change

- Update action examples to install svn

Closes https://github.com/10up/action-wordpress-plugin-deploy/issues/151

### Checklist:
- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [x] I have updated the documentation accordingly.
- [x] I have added [Critical Flows, Test Cases, and/or End-to-End Tests](https://10up.github.io/Open-Source-Best-Practices/testing/) to cover my change.
- [x] All new and existing tests pass.

### CODE REVIEW

File: examples/deploy-on-publishing-a-new-release-and-attach-a-zip-file-to-the-release.yml [hunk 1]
```diff
@@ -54,6 +54,10 @@ jobs:
       run: |
         npm install
         npm run build
+
+    - name: Install SVN ( Subversion )
+      run: sudo apt-get install subversion
```
Comment (iamdharmesh) [inline examples/deploy-on-publishing-a-new-release-and-attach-a-zip-file-to-the-release.yml]: It is better to run `sudo apt-get update` before installing a package. As suggested in a note [here](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/customizing-github-hosted-runners#installing-software-on-ubuntu-runners:~:text=Always%20run%20sudo%20apt%2Dget%20update%20before%20installing%20a%20package.%20In%20case%20the%20apt%20index%20is%20stale%2C%20this%20command%20fetches%20and%20re%2Dindexes%20any%20available%20packages%2C%20which%20helps%20prevent%20package%20installation%20failures.).

```suggestion
    - name: Install SVN ( Subversion )
      run: |
        sudo apt-get update
        sudo apt-get install subversion
```

File: examples/deploy-on-pushing-a-new-tag-and-create-release-with-attached-zip.yml [hunk 2]
```diff
@@ -14,6 +14,8 @@ jobs:
       run: |
         npm install
         npm run build
+    - name: Install SVN ( Subversion )
+      run: sudo apt-get install subversion
```
Comment (iamdharmesh) [inline examples/deploy-on-pushing-a-new-tag-and-create-release-with-attached-zip.yml:18]: Same as above.

File: examples/deploy-on-pushing-a-new-tag.yml [hunk 3]
```diff
@@ -13,6 +13,8 @@ jobs:
       run: |
         npm install
         npm run build
+    - name: Install SVN ( Subversion )
+      run: sudo apt-get install subversion
```
Comment (iamdharmesh) [inline examples/deploy-on-pushing-a-new-tag.yml:17]: Same as above.

**Review by iamdharmesh — COMMENTED** [review]
Hi @thrijith, 

Thanks a lot for the PR, it looks good to me. Just a minor suggestion around installing the package, based on GH action docs. Could you please check it once?

Thank you.

File: examples/deploy-on-pushing-a-new-tag-and-create-release-with-attached-zip.yml (hunk 2 above)
Comment (iamdharmesh) [inline examples/deploy-on-pushing-a-new-tag-and-create-release-with-attached-zip.yml]:
```suggestion
    - name: Install SVN ( Subversion )
      run: |
        sudo apt-get update
        sudo apt-get install subversion
```

File: examples/deploy-on-pushing-a-new-tag.yml (hunk 3 above)
Comment (iamdharmesh) [inline examples/deploy-on-pushing-a-new-tag.yml]:
```suggestion
    - name: Install SVN ( Subversion )
      run: |
        sudo apt-get update
        sudo apt-get install subversion
```

**Review by iamdharmesh — APPROVED** [review]
LGTM, Thanks @thrijith. @dkotter Could you please help with a quick check and merge as I have made minor change to the PR.

**Review by dkotter — APPROVED** [review]

**Pull request merged**